scripts/prometheus-dev.py

```diff
@@ -145,16 +145,16 @@ def run_pushgateway(container_name, network_name):
 
     prometheus_config = os.path.abspath(prometheus_config)
 
-    # Checked if passed ekiden container is running
-    if not container_running(ekiden_container):
-        print("ERROR: Ekiden container: '{}' not running!".format(ekiden_container))
+    # Checked if passed Oasis node container is running
+    if not container_running(oasis_node_container):
```

Comment:
Is this script still useful or did you just do a general rename? This script expects that oasis-node was run in a docker container (from old times) which is not the case for most users, so this check should at least be made optional (also the part where oasis-node and prometheus networks are connected).

However if the script was not useful you can probably just delete it.

Reply:
I just did a general rename as I thought we're still using it somewhere. I removed the whole script now.